composer.json

```diff
@@ -7,6 +7,8 @@
     "require": {
         "php": "^7.4",
         "ext-json": "*",
+        "dusterio/lumen-passport": "^0.3.0",
```

Comment:
Hi,

I think you recommended the use of Passport.

_"we should consider use Passport for this. It may come with a lot more than we need but the alternative of rolling out our own authentication system isn't ideal. Unfortunately, AFAIK, Sanctum isn't supported on Lumen. 🤷‍♂️"_

However due to the research I did it is not possible to install Passport on Lumen.

As it is the first time that I'm using Lumen, I think it is necessary to install this external package.

 Do you have any alternative suggestions?

Reply:
Yes, I know, and I'm starting to think it may have been a mistake :thinking: 

We should discuss other implementations. The requirements are that the user exchanges their credentials for a token associated with the device they log in. That token should have an expiration date, too.

Any ideas? My first though is that we could use the default authentication service that Lumen supports. Let's discuss this further in our Discord server.